### DISCUSSION TITLE
`pipelines/_utils.py` custom tags

### DISCUSSION THREAD

**Discussion by mithunshukla** [aws-samples/mlops-sagemaker-github-actions#8] [Q&A] [answered]
Hi there,
In the `get_pipeline_custom_tags` function within the `https://github.com/aws-samples/mlops-sagemaker-github-actions/blob/main/seedcode/pipelines/_utils.py` module, is it just me or is there some room for improvement?

I can see there are some issues have been raised for that too.

**Comment by pooyavahidi** [comment, marked as answer]
Hi @mithunshukla, yes, I agree. I think the the error handling is somewhat incomplete. It has the `try except` block but it doesn't actually raise the error, and also it prints as "error" message which is misleading for user. I looked at the closed issue and that was exactly the mislead the user.

Please raise a PR if you have a chance and I'll merge it.
Thanks

**Reply by mithunshukla** [reply]
Cool, I'll do that.

**Reply by pooyavahidi** [reply]
Awesome! thanks.
